### PR TITLE
changes to add qREGEN

### PR DESCRIPTION
**Pull request by prakritimalik**

### CODE REVIEW

**Comment by prakritimalik** [comment]
> @prakritimalik is attempting to deploy a commit to the **OsmoLabs** Team on [Vercel](https://vercel.com).
> 
> A member of the Team first needs to [authorize it](https://vercel.com/git/authorize?team=OsmoLabs&type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%225ed65530f6c4868830c2da742483083622797798%22%7D%2C%22id%22%3A%22QmWff1zmvH3QyNtc9PfT6yVincahCSoZcymwXt6BjbhWEw%22%2C%22org%22%3A%22osmosis-labs%22%2C%22prId%22%3A1385%2C%22repo%22%3A%22osmosis-frontend%22%7D).

@JeremyParish69 I raised a PR to add qREGEN in the assetlist json files (https://github.com/osmosis-labs/assetlists/pull/402/files) but I'm not able to set 
  "pools": {
        "REGEN": 948
      },
      
      getting this error: Property REGEN is not allowed
      
      Can you please help me with this?

**Comment by prakritimalik** [comment]
> @prakritimalik is attempting to deploy a commit to the **OsmoLabs** Team on [Vercel](https://vercel.com).
> 
> A member of the Team first needs to [authorize it](https://vercel.com/git/authorize?team=OsmoLabs&type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%225ed65530f6c4868830c2da742483083622797798%22%7D%2C%22id%22%3A%22QmWff1zmvH3QyNtc9PfT6yVincahCSoZcymwXt6BjbhWEw%22%2C%22org%22%3A%22osmosis-labs%22%2C%22prId%22%3A1385%2C%22repo%22%3A%22osmosis-frontend%22%7D).

@JeremyParish69 I raised a PR to add qREGEN in the assetlist json files (https://github.com/osmosis-labs/assetlists/pull/402/files) but I'm not able to set
```
"pools": {
"REGEN": 948
},
```

  getting this error: Property REGEN is not allowed
  
  Can you please help me with this?

File: packages/web/config/price.ts [hunk 1]
```diff
@@ -1149,6 +1149,19 @@ export const PoolPriceRoutes: IntermediateRoute[] = [
     ),
     destCoinId: "pool:uatom",
   },
+  {
+    alternativeCoinId: "pool:uqregen",
+    poolId: "948",
+    spotPriceSourceDenom: DenomHelper.ibcDenom(
+      [{ portId: "transfer", channelId: "channel-522" }],
+      "uqatom"
```
Comment (JeremyParish69) [inline packages/web/config/price.ts:1157]: these still have the min denom for qatom and atom rather than qregen and regen

**Comment by JeremyParish69** [comment]
This branch was taken before the merging of the previous asset, so there are some merge conflicts to be resolved.

**Comment by prakritimalik** [comment]
> This branch was taken before the merging of the previous asset, so there are some merge conflicts to be resolved.

Merge conflicts have been resolved.

**Review by JeremyParish69 — APPROVED** [review]

**Pull request merged**